packages/api-plugin-fulfillment/src/mutations/updateFulfillmentOptionsForGroup.js

```diff
@@ -0,0 +1,102 @@
+import _ from "lodash";
+import SimpleSchema from "simpl-schema";
+import ReactionError from "@reactioncommerce/reaction-error";
+import getCartById from "../util/getCartById.js";
+
+const inputSchema = new SimpleSchema({
+  cartId: String,
+  cartToken: {
+    type: String,
+    optional: true
+  },
+  fulfillmentGroupId: String
+});
+
+/**
+ * @name getShipmentQuotesQueryStatus
```

Comment:
Don't really know what format these JSDocs are in, but they are not consistent with the JSDocs we do everywhere else

Reply:
It was copied over from old shipment plugin. Fixed it.